tools/cdk-release/test/changelog.test.ts

```diff
@@ -0,0 +1,106 @@
+import { ConventionalCommit } from '../lib/conventional-commits';
+import { changelog } from '../lib/lifecycles/changelog';
+import { ReleaseOptions } from '../lib/types';
+
+describe('Changelog generation', () => {
+  const args: ReleaseOptions = {
+    releaseAs: 'minor',
+    dryRun: true,
+    silent: true,
+  };
+
+  test("correctly handles 'BREAKING CHANGES'", async () => {
+    const commits: ConventionalCommit[] = [
+      buildCommit({
+        type: 'feat',
+        subject: 'super important feature',
+        notes: [
+          {
+            title: 'BREAKING CHANGE',
+            text: 'this is a breaking change',
+          },
+        ],
+      }),
+      buildCommit({
+        type: 'fix',
+        scope: 'scope',
+        subject: 'hairy bugfix',
+      }),
+      buildCommit({
+        type: 'chore',
+        subject: 'this commit should not be rendered in the Changelog',
+      }),
+    ];
+
+    const changelogContents = await changelog(args, '1.23.0', '1.24.0', commits);
+
+    expect(changelogContents).toBe(
+      `## [1.24.0](https://github.com/aws/aws-cdk/compare/v1.23.0...v1.24.0) (2021-06-28)
```

Comment:
Oh this is funny. I didn't realize this date was here. This test will fail tomorrow 😃.